diff --git a/cjs/react-server-dom-webpack-client.browser.development.js b/cjs/react-server-dom-webpack-client.browser.development.js
index 38e04fbe3d8aae7538ff51a742fa5fe683dbad90..e1f8d26157543aff1abebfde0408e97979bff90b 100644
--- a/cjs/react-server-dom-webpack-client.browser.development.js
+++ b/cjs/react-server-dom-webpack-client.browser.development.js
@@ -11,6 +11,7 @@
 "use strict";
 "production" !== process.env.NODE_ENV &&
   (function () {
+    globalThis.__webpack_require__ = (id) => import(id);
     function _defineProperty(obj, key, value) {
       key in obj
         ? Object.defineProperty(obj, key, {
diff --git a/cjs/react-server-dom-webpack-client.browser.production.js b/cjs/react-server-dom-webpack-client.browser.production.js
index 7a5db2b4f5f9668bea617921595c490c18a13c7a..fe00756f3fc4d8cfe92ffd96ab68b21037f984f7 100644
--- a/cjs/react-server-dom-webpack-client.browser.production.js
+++ b/cjs/react-server-dom-webpack-client.browser.production.js
@@ -11,6 +11,9 @@
 "use strict";
 var ReactDOM = require("react-dom"),
   decoderOptions = { stream: !0 };
+
+globalThis.__webpack_require__ = (id) => import(id);
+
 function resolveClientReference(bundlerConfig, metadata) {
   if (bundlerConfig) {
     var moduleExports = bundlerConfig[metadata[0]];
diff --git a/cjs/react-server-dom-webpack-server.browser.development.js b/cjs/react-server-dom-webpack-server.browser.development.js
index 77b814b6e906cc699e9044c8994657cee9dfb0ae..5e10e4db0a765e6d2ee3e8b7ec0644d4565f9272 100644
--- a/cjs/react-server-dom-webpack-server.browser.development.js
+++ b/cjs/react-server-dom-webpack-server.browser.development.js
@@ -11,6 +11,8 @@
 "use strict";
 "production" !== process.env.NODE_ENV &&
   (function () {
+    globalThis.__webpack_require__ = (id) => import(id);
+
     function voidHandler() {}
     function _defineProperty(obj, key, value) {
       key in obj
diff --git a/cjs/react-server-dom-webpack-server.browser.production.js b/cjs/react-server-dom-webpack-server.browser.production.js
index 0f4e7e1b00495366dff3646ae07fd2cb12ed16b7..6a4fce27e1f2ac8c563d30a0ec2f588d53195d27 100644
--- a/cjs/react-server-dom-webpack-server.browser.production.js
+++ b/cjs/react-server-dom-webpack-server.browser.production.js
@@ -13,6 +13,8 @@ var ReactDOM = require("react-dom"),
   React = require("react"),
   channel = new MessageChannel(),
   taskQueue = [];
+globalThis.__webpack_require__ = (id) => import(id);
+
 channel.port1.onmessage = function () {
   var task = taskQueue.shift();
   task && task();
