diff --git a/dist/node/chunks/dep-Cg8OuIew.js b/dist/node/chunks/dep-Cg8OuIew.js
index 89766f332b458ed06e4cf7e79f1f7c91494dd394..4e86a257d2af1c701939debf48f387fe46cdf0ef 100644
--- a/dist/node/chunks/dep-Cg8OuIew.js
+++ b/dist/node/chunks/dep-Cg8OuIew.js
@@ -14311,6 +14311,7 @@ function parentDir(p) {
 /** @type {import('./index').LoaderSync} */
 const jsonLoader = (_, content) => JSON.parse(content);
 // Use plain require in webpack context for dynamic import
+const __non_webpack_require__ = __require
 const requireFunc =
 	typeof __webpack_require__ === 'function' ? __non_webpack_require__ : __require;
 /** @type {import('./index').LoadersSync} */
